pkg/controller/controller.go

```diff
@@ -329,7 +329,6 @@ func (c *Controller) bootWithError(ctx context.Context) error {
 		for {
 			resetWait := c.resyncPeriod * 4
 			time.Sleep(resetWait)
-			errorGauge.Set(0)
```

Comment:
We have alerts with `rate()` so there's no need to set it to `0`. With labels this also doesn't make sense anymore.